src/webui/webapplication.cpp

```diff
@@ -65,6 +65,7 @@
 const int MAX_ALLOWED_FILESIZE = 10 * 1024 * 1024;
 const auto C_SID = QByteArrayLiteral("SID"); // name of session id cookie
 
+const QString HEADER_API_TOKEN = u"x-api-token"_qs; // name of api token header
```

Comment:
I think the standard `Authorization` header would be more appropriate here, with a scheme of `Bearer`.

Reply:
Switched to `Bearer` scheme.